### PR TITLE
Read the moves at stages 'join' and 'catch' to be able to better identify pokemons

### PR DESCRIPTION
**Pull request by pifopi**

### CODE REVIEW

File: automaxlair/MaxLairInstance.py [hunk 1]
```diff
@@ -294,23 +350,44 @@ def read_selectable_pokemon(self, stage: str) -> List[Pokemon]:
...
+            move_3 = self.read_text(image, self.move_rect_15, threshold=False, segmentation_mode='--psm 7').strip()
+            move_4 = self.read_text(image, self.move_rect_16, threshold=False, segmentation_mode='--psm 7').strip()
+            moves.append(move_1 + move_2 + move_3 + move_4)
         elif stage == 'battle':
             pokemon_names.append(self.read_text(image, self.sel_rect_5, threshold=False, invert=False, segmentation_mode='--psm 8').strip())
             abilities.append('')
             type_1 = self.read_text(image, self.type_rect_1, threshold=False, invert=True, segmentation_mode='--psm 8').strip().title()
             type_2 = self.read_text(image, self.type_rect_2, threshold=False, invert=True, segmentation_mode='--psm 8').strip().title()
             types.append(type_1 + type_2)
+            moves = ['','','']
```
Comment (ercdndrs) [inline automaxlair/MaxLairInstance.py]: Should there be 4 moves here?

**Review by ercdndrs — CHANGES_REQUESTED** [review]
See my comment about changing the length of the empty move list.

File: automaxlair/MaxLairInstance.py (hunk 1 above)
Comment (pifopi) [reply automaxlair/MaxLairInstance.py]: In fact, it should be that. We're not giving the moves one by one but by concatenating them (just like types)

Comment (ercdndrs) [reply automaxlair/MaxLairInstance.py]: Ah I understand now. Looks good!

**Review by ercdndrs — APPROVED** [review]

**Pull request merged**